packages/cozy-stack-client/src/AppsRegistryCollection.js

```diff
@@ -1,13 +1,22 @@
 import { transformRegistryFormatToStackFormat } from 'cozy-client/dist/registry'
 
 import DocumentCollection, { normalizeDoc } from './DocumentCollection'
-import { FetchError } from './errors'
 
 export const APPS_REGISTRY_DOCTYPE = 'io.cozy.apps_registry'
 
 export const normalizeAppFromRegistry = (app, doctype) => {
+  // The stack returns some app without an id, so we use the slug as id.
+  // Sometime we use latest_version.manifest.source as an id.
```

Comment:
That means there is 2 levels of normalization for the registry? Shouldn't we only use `normalizeAppFromRegistry`?

Reply:
After a discussion with @Crash--, we came to the same conclusion: this new version has only one normalization left